.github/workflows/codeql-analysis.yml

```diff
@@ -1,62 +1,61 @@
-name: "CodeQL"
+name: "Code Scanning - Action"
 
 on:
   push:
     branches: [master]
```

Comment:
Shouldn't `develop` also be added here? I have no idea myself, just thought so based on the old comment:
![image](https://user-images.githubusercontent.com/43886029/233676195-95d913a6-53e5-4569-9516-0658fe484a25.png)